### PR TITLE
kubectl: add container ports in pod description

### PR DESCRIPTION
**Pull request by 0xmichalis**
@smarterclayton @kubernetes/kubectl PTAL

### CODE REVIEW

File: pkg/kubectl/describe.go [hunk 1]
```diff
@@ -795,6 +802,14 @@ func DescribeContainers(containers []api.Container, containerStatuses []api.Cont
 	}
 }
 
+func describeContainerPorts(cPorts []api.ContainerPort) string {
+	ports := []string{}
+	for _, cPort := range cPorts {
+		ports = append(ports, strconv.Itoa(cPort.ContainerPort))
```
Comment (smarterclayton) [inline pkg/kubectl/describe.go]: We probably need to describe TCP vs UDP

Comment (kargakis) [inline pkg/kubectl/describe.go]: parenthesized the port protocol, PTAL

**Comment by smarterclayton** [comment]
Lgtm, please resolve whether this is a test flake and if not I'll give the label when I land

**Comment by kargakis** [comment]
Seems like a flake but I cannot determine the origin of it.

**Comment by bgrant0607** [comment]
Example output?

**Comment by bgrant0607** [comment]
```
Error deleting container: Error response from daemon: Conflict, You cannot remove a running container. Stop the container before attempting removal or use -f
Unrecognized input header
```

@k8s-bot unit test this issue: #21451

**Comment by kargakis** [comment]
Adds last line only in the following output. Omitted the rest of it since the description of a pod is verbose.
```
Name:		docker-registry-2-esq78
Namespace:	default
Image(s):	openshift/origin-docker-registry:latest
Node:		localhost.localdomain/10.0.2.15
Start Time:	Mon, 22 Feb 2016 11:35:57 +0000
Labels:		deployment=docker-registry-2,deploymentconfig=docker-registry,docker-registry=default
Status:		Running
Reason:		
Message:	
IP:		172.17.0.4
Controllers:	ReplicationController/docker-registry-2
Containers:
  registry:
    Container ID:	docker://525cafae3acaa034544da95135072c76452ac19146fccc14520b62173027362c
    Image:		openshift/origin-docker-registry:latest
    Image ID:		docker://6c69711151868bda91e1ff3d7e12418f52bffdc854ecb4021804ad35ba3519ee
    Ports:		5000 (TCP), 6001 (TCP)
```

**Comment by thockin** [comment]
`describe service` uses `31812/TCP` syntax - we should be consistent.  One or the other should change

**Comment by smarterclayton** [comment]
Tim's right.  Let's use /TCP for now since that matches describe and
Dockerfile.

**Comment by kargakis** [comment]
updated to use the port/protocol syntax

**Comment by kargakis** [comment]
rebased on top of head, @smarterclayton please relabel this

**Pull request merged**